test/test_type_info.py

```diff
@@ -14,17 +14,17 @@
 class TestDTypeInfo(TestCase):
 
     def test_invalid_input(self):
-        for dtype in [torch.float32, torch.float64]:
+        for dtype in [torch.float16, torch.float32, torch.float64]:
```

Comment:
Should torch.bfloat16, torch.complex64, and torch.complex128 be added to this list?

Reply:
Also torch.bool?

Reply:
Addressed!